packages/minter-contracts/ligo/src/swaps/Makefile

```diff
@@ -18,10 +18,15 @@ MODULES ?= ../..
 BIN ?= ../../../bin
 
 all: \
-	$(BIN)/fa2_swap.tz
+	$(BIN)/fa2_swap.tz \
+	$(BIN)/fa2_whitelisted_swap.tz
```

Comment:
Catching this a little bit late unfortunately, but we have an existing [solution in place for compiling contracts](https://github.com/tqtezos/minter-sdk/blob/main/packages/minter-contracts/src/compile-contracts.ts#L9). We should stay with a single compilation process to avoid fragmentation moving forward. Could you update this to hook into our existing build process? I'll send up an additional PR as well for handling the `fa2_swap` as well.

Reply:
Yep, I update that script in this PR [too](https://github.com/tqtezos/minter-sdk/pull/49/files#diff-08abcafcc8045eae61df04d7876cabce7fcadf7037bb45ad025ff7f4cfa55eedR16).

It's just that I need a way to run Haskell tests, preferably along with LIGO contracts compilation, and doing this from TS script does not seem proper. Plus, there should be a way to compile just one contract quickly, right? Optionally passing some flags like `-werror` once available.

Though I agree that having two different solutions for this does not sound good as well. What are your thoughts on this?

Reply:
I'm trying out a CI solution here — https://github.com/tqtezos/minter-sdk/pull/78

It has a discrete job which compiles all of the LIGO contracts and generates an artifact which any other job may reference. As such, the other jobs may just "assume" that the contracts have been compiled. Certainly not quite finalized yet, but what do you think about this?

Reply:
https://github.com/tqtezos/minter-sdk/pull/81

Reply:
That sounds quite good :+1:

Then let's handle this like: you remove this `swaps/Makefile` in #81 since it is not necessary for CI there anymore, and then which PR gets merged first does not really matter.

Reply:
@zachrbrown I rebased and this Makefile is gone. Any objections if I merge?

Reply:
Just one small issue https://github.com/tqtezos/minter-sdk/pull/49#discussion_r603417616